tools/branch.info

```diff
@@ -0,0 +1 @@
+edge_features
```

Comment:
What is this file? Can we remove it?

Reply:
Done. This was some debug file I used to track my changes across branches.